.eslintrc.js

```diff
@@ -1,32 +1,34 @@
 module.exports = {
-  env: {
-    browser: true,
-    es2021: true,
-    jest: true,
-  },
-  extends: [
-    'eslint:recommended',
```

Comment:
is it needed if we have this? `'plugin:@typescript-eslint/recommended'`

Reply:
These are `eslint:recommended` rules: https://github.com/eslint/eslint/blob/main/conf/eslint-recommended.js
And these are `plugin:@typescript-eslint/recommended` rules: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/eslint-recommended.ts

`eslint:recommended` looks more generic and it looks like it's enabling more rules.

Also note the comment at the top of typescript one:
> - disables rules from eslint:recommended which are already handled by TypeScript.

So it overrides some rules of `eslint:recommended`. The order of them seems important too. We should specify the `eslint:recommended` first and then `plugin:@typescript-eslint/recommended`.

Reply:
Ah, also documentation mentiones that they need to be used together: https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/CONFIGS.md#eslint-recommended

Reply:
I don't see `estlint:recommended` has been added back as nazim suggested, it would be good to do it in a follow-up :-)

Reply:
> I don't see `estlint:recommended` has been added back as nazim suggested, it would be good to do it in a follow-up :-)

ah, thanks for catching that. I thought I had added it.